src/keyword.rs

```diff
@@ -19,7 +18,7 @@ pub struct KeywordScore {
 
 impl KeywordScore {
     /// Creates a vector of `KeywordScore` from `mp`
-    pub fn from_map(mp: HashMap<String, f64>) -> Vec<Self> {
+    pub fn from_map(mp: impl IntoIterator<Item = (String, f64)>) -> Vec<Self> {
```

Comment:
I think it's better to implement `std::convert::From<T: IntoIterator>` trait for `KeywordScore`. what is your idea?

Reply:
That would be fine too. I just kept this one for backwards compatibility.

Reply:
you are right, however, since this library is in beta state, I think we could break backwards compatibility.